modules/appeals_api/app/mailers/appeals_api/weekly_error_report_mailer.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'appeals_api/decision_review_report'
+
+module AppealsApi
+  class WeeklyErrorReportMailer < ApplicationMailer
+    def build(date_from:, date_to:, recipients:, friendly_duration: '')
+      @report = DecisionReviewReport.new(from: date_from, to: date_to)
+      @friendly_duration = friendly_duration
+      @friendly_env = (Settings.vsp_environment || Rails.env).titleize
+
+      body = []
+      body += stuck_err_hlrs
```

Comment:
Since this is essentially a CSV, let's add a header line that details what each column is